package.json

```diff
@@ -212,13 +212,13 @@
     "start": "concurrently --kill-others --prefix=name --names=SERVER,MOCK --prefix-colors=cyan,reset --passthrough-arguments 'yarn dev-server --env noproxy {@}' 'mockdata/mockserver.py' --",
     "start:fec": "concurrently --kill-others --prefix=name --names=SERVER,MOCK --prefix-colors=cyan,reset --passthrough-arguments 'HOT=true yarn dev:fec --clouddotEnv prod {@}' 'mockdata/mockserver.py' --",
     "build": "webpack $(run/progress-flag-if-tty.sh)",
-    "build:prod": "yarn build --mode=production",
-    "build:saas": ". run/sentry-env.sh && yarn build:prod --output-path 'dist'",
+    "build:prod": "MODE=prod yarn build:fec",
```

Comment:
i feel since we now use a standard command to run fec (i.e. `build:prod`), this is essentially the turning point where we officially 'switch' - in both dev-envs and CI.

and since we're 'officially' switching, we should probably also switch up the command names to reflect that:

`command:fec` --> `command`
`command` --> `command:webpack`

Reply:
actually that ☝️  just complicates things..

on second thought - why aren't we just removing the old commands altogether at this point?

that way we can remain with the bare command names (i.e. ditch the `:fec` suffixes)

and while we're at it - why not remove webpack.config, too?  it's not used anyway 🤔